.github/workflows/ci-build.yml

```diff
@@ -26,12 +26,24 @@ jobs:
       with:
         path: Meadow.Core # we have to put this in a sub-folder so we can pull Contracts/Units to a parent folder
        
+    - name: Determine Meadow.Contracts branch to use
```

Comment:
Thoughts on this? It attempts to check out Meadow.Contracts based on the same branch name as the the branch name for the PR for Meadow.Core. If it doesn't exist, it defaults to `develop`. I can totally get rid of it if it's easier to keep it the way it is.

Reply:
we have a chain of dependencies, so if we do it for Contracts, we should do it for all?